index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="UTF-8">
+		<title>Touch Bar Simulator</title>
+		<meta name="description" content="The macOS Touch Bar Simulator as a standalone app">
+		<meta name="author" content="Sindre Sorhus">
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+		<meta property="og:url" content="https://github.com/sindresorhus/touch-bar-simulator">
+		<meta property="og:title" content="Touch Bar Simulator">
+		<meta property="og:description" content="The macOS Touch Bar Simulator as a standalone app">
+		<meta property="og:image" content="https://sindresorhus.com/media/touch-bar-simulator.png">
```

Comment:
Can you add Twitter ones too? Same as https://github.com/sindresorhus/caprine/blob/615ac800b3fe3be3d8521a78743e07a179c72b05/index.html#L10-L14